tests/pytests/test_acl.py

```diff
@@ -193,10 +218,13 @@ def test_internal_commands(env):
 
     # Now `test` should not be able to execute RediSearch internal commands
     # `_FT.DEBUG` has only subcommands, so we check it separately.
-    internal_commands = INTERNAL_SEARCH_COMMANDS[::]
-    internal_commands.remove('_FT.DEBUG')
-    for command in internal_commands:
-        env.expect(command).error().contains("User test has no permissions to run")
+    internal_commands = copy.deepcopy(INTERNAL_SEARCH_COMMANDS)
```

Comment:
Why the deep copy? Not needed here

Reply:
we remove `_FT.DEBUG` from the container.
Assumed it might be needed in other functions.

Reply:
A shallow copy will work as well, e.g., via the `dict.copy()` or `dict(dict_to_copy)` APIs.